presto-main/src/main/java/com/facebook/presto/sql/analyzer/Scope.java

```diff
@@ -31,7 +31,9 @@
 import static com.facebook.presto.sql.analyzer.SemanticExceptions.missingAttributeException;
```

Comment:
> Parent scope WithQueries should always be visible from
> child scope even though parent fields might not.

In what case would fields from the parent scope not be visible in the child scope?